astoria/common/metadata.py

```diff
@@ -32,15 +34,37 @@ def init(cls, config: AstoriaConfig) -> 'Metadata':
         Based on information from software and the astoria configuration.
         """
         uname = platform.uname()
+        os_release = cls.get_os_version_info()
         return cls(
             astoria_version=__version__,
             kernel_version=uname.release,
             arch=uname.machine,
             python_version=platform.python_version(),
             libc_ver="".join(platform.libc_ver()),
             usercode_entrypoint=config.astprocd.default_usercode_entrypoint,
+            os_name=os_release['NAME'],
+            os_pretty_name=os_release['PRETTY_NAME'],
+            os_version=os_release['VERSION_ID'],
```

Comment:
What happens if these keys don't exist? We should probably use dict.get instead of letting the KeyError be thrown.